src/views/Records/Records.vue

```diff
@@ -89,16 +71,18 @@ export default {
     recordTypes: recordsLabels['recordTypes']
   }),
   computed: {
-    ...mapState('uiController', ['scrollStatus','stickToTop']),
+    ...mapState('uiController', ['scrollStatus', 'stickToTop']),
     ...mapState('records', ['records']),
     getTitle: function () {
       const flipRecordTypes = Object.entries(this.recordTypes).reduce(
           (obj, [key, value]) => ({...obj, [value]: key}),
           {}
       );
       let title = "Search";
-      if (Object.prototype.hasOwnProperty.call(this.$route.query, 'fairsharingRegistry')) {
-        if (Object.prototype.hasOwnProperty.call(flipRecordTypes, this.$route.query.fairsharingRegistry)) {
+      if (Object.prototype.hasOwnProperty.call(this.$route.query, 'fairsharingRegistry'))
+      {
```

Comment:
Thanks for spotting this! but I believe this has been mistakenly copy-pasted from my notepad and caused the issue. it will be fixed by the next commits.